plan/entities.go

```diff
@@ -81,6 +87,7 @@ type TestCase struct {
 	Plan      *Plan
 	Client    string
 	Arguments TestClientArgs
+	Skip      bool
```

Comment:
Yep good idea. Also added a naive message saying filtered out due to pruning. We can beef it up if needed.